app/planet.py

```diff
@@ -0,0 +1,19 @@
+class Planet:
```

Comment:
Good work separating this out into it's own function. When we learn about models, we will organize the model classes in a director called `models`.